lib/common.sh

```diff
@@ -32,7 +54,7 @@ _log()
     echo -e "${color}${msg}${NORMAL}"
     (
         flock -e 999
-        echo -e "$(date -u +"%a %b %d %G %T.%3N") $HOSTNAME $$: ${color}${msg}${NORMAL}" >> $LOGFILE
+        echo -e "${prefix}: $(date -u +"%a %b %d %G %T.%3N") $HOSTNAME $$: ${color}${msg}${NORMAL}" >> $LOGFILE
```

Comment:
there should be no colon and space after ${prefix}. Note that we already added a space at the end in $prefix.
Also a colon will not look good in the case where ${prefix} is an empty string.
echo -e "${prefix}: $(date..." should be 
echo -e "${prefix}$(date..." 

Same for L128